lib/generators/templates/coupons.rb

```diff
@@ -7,8 +7,7 @@
 # Stripe::Coupons::Gold25 #=> 'gold25'
 
 # Stripe.coupon :gold25 do |coupon|
-#
-#   # specify if this coupon is useable 'once', 'forever', or 'repeating
+#   # specify if this coupon is usable 'once', 'forever', or 'repeating'
```

Comment:
"useable" seems to be a known variant https://english.stackexchange.com/a/69710 . Let's keep it!

Reply:
@tansengming, sure thing, done!